pkg/network/kube_proxy_test.go

```diff
@@ -98,6 +127,66 @@ winkernel:
 `))
 }
 
+func TestKubeProxyIPv6Config(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	errs := validateKubeProxy(&configIPv6)
+	g.Expect(errs).To(HaveLen(0))
+
+	cfg, err := kubeProxyConfiguration(map[string]operv1.ProxyArgumentList{
+		// special address+port combo
+		"metrics-bind-address":   {"fd00:1234::4"},
+		"metrics-port":           {"999"},
+		"conntrack-max-per-core": {"10"},
+	},
+		&configIPv6,
```

Comment:
I don't like the way this indents; can you add a newline before `map` so the rest of the map body up to the `},` indents another level?

Reply:
done,